modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -320,51 +320,60 @@ const PolicyDetail = ({
 											<div className="align-items-center d-flex data-panel-hide float-left justify-content-between w-25">
 												{item.date}
 											</div>
-											<Panel
-												Description={() =>
-													ContentDescription(
-														item.description
-													)
-												}
-												key={index}
-												setShowPanel={() =>
-													displayHistoryPanel(index)
-												}
-												showPanel={showPanel[index]}
-											>
-												<div className="justify-content-between layout-show-details ml-auto mt-4 pb-3 pl-3 pr-3 pt-3 w-75">
-													<div className="d-flex flex-row justify-content-between">
-														<div className="align-self-start mt-2">
-															<p className="mb-1 text-neutral-7 w-25">
-																Amount
-															</p>
-															<div>
-																{
-																	item.detail_Amount
-																}
+											<div className="w-100">
+												<Panel
+													Description={
+														<ContentDescription
+															description={
+																item.description
+															}
+														/>
+													}
+													hasExpandedButton={true}
+													isPanelExpanded={
+														showPanel[index]
+													}
+													key={index}
+													setIsPanelExpanded={() =>
+														displayHistoryPanel(
+															index
+														)
+													}
+												>
+													<div className="justify-content-between layout-show-details ml-auto mt-4 p-3 w-75">
+														<div className="d-flex flex-row justify-content-between">
+															<div className="align-self-start mt-2">
+																<p className="mb-1 text-neutral-7 w-25">
+																	Amount
+																</p>
+																<div>
+																	{
+																		item.detail_Amount
+																	}
+																</div>
+															</div>
+															<div className="align-self-start mt-2 w-50">
+																<p className="mb-1 text-neutral-7">
+																	Injuries Or
+																	Fatalities
+																</p>
+																<div>
+																	{
+																		item.detail_Injuries
+																	}
+																</div>
```

Comment:
Maybe you should componentize this.

Reply:
This is a children of the panel component.